concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1286,10 +1286,6 @@ jobs:
                 . terraform-variables/cf-secrets.tfvars.sh
 
                 cp cf-tfstate/cf.tfstate updated-tfstate/cf.tfstate
-                terraform import aws_s3_bucket_lifecycle_configuration.droplets-s3 ((deploy_env))-cf-droplets
```

Comment:
Why did we add and remove this in a single PR?

Reply:
It wasn't actually needed but was referenced in the upgrade documents